### PR TITLE
add v4l-utils rhel key

### PR DESCRIPTION
**Pull request by flynneva**
`v4l-utils` was missing a key for `rhel` so just adding it in as discussed over at [this PR for the `usb_cam` package](https://github.com/ros/rosdistro/pull/30306).

https://rpms.remirepo.net/rpmphp/zoom.php?rpm=v4l-utils

### CODE REVIEW

**Comment by flynneva** [comment]
not sure why the nose-tests are failing....maybe `v4l-utils` should be `v4l-devel` for rhel instead?

**Comment by flynneva** [comment]
so it looks like in the `rosdep/base.yaml` file `v4l` is listed twice? once for the `libv4l-devel` package and the other just for `v4l-utils` package. im guessing the devel one also includes the `v4l-utils`.

**Comment by cottsay** [comment]
I pretty sure that `v4l-utils` is not packaged for RHEL 8 at the moment - at least not in any repositories that rosdep targets. It is packaged for RHEL 7, but that won't help you with `usb_cam`.

Here's the RHEL 7 package: http://mirror.centos.org/centos/7/os/x86_64/Packages/v4l-utils-0.9.5-4.el7.x86_64.rpm

`libv4l-devel` doesn't proved the same utilities that `v4l-utils` does, so that's not an appropriate package to satisfy this key.

So this is the only RHEL rule for `v4l-utils` that would be correct right now, and like I said, it won't help when Blooming `usb_cam`.
```yaml
  rhel:
    '7': [v4l-utils]
```

**Comment by flynneva** [comment]
@cottsay  ah that makes sense. so maybe ill just close this then, or do you think there would be any value with adding only the rhel 7 `v4l-utils` key?

**Review by tfoote — APPROVED** [review]

**Review by mabelzhang — APPROVED** [review]

**Comment by mabelzhang** [comment]
@flynneva I guess this is your call, since it's your package that prompted you to add this. Do you want it? We can merge this if you still want it. Otherwise you can close it.

**Comment by flynneva** [comment]
@mabelzhang I mean, sure? If it won't hurt anyone by merging I think it'd be good to at least show in rosdep what version of rhel is supported?

Not a high priority for me though honestly so if it's better just to wait until v4l is available for rhel 8 as well then that works too

**Pull request merged**
